hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -458,6 +457,15 @@ public boolean scheduleExists(ScheduleID id) {
 		return Optional.of(info.build());
 	}
 
+	public Optional<List<TokenNftInfo>> infoForAccountNft(AccountID id, long start, long end) {
+		var account = accounts().get(fromAccountId(id));
+		if (account == null) {
+			return Optional.empty();
+		}
+
+		return Optional.of(tokenStore.getUniqueTokenInfosFor(id, start, end));
```

Comment:
normally the token store returns merkle objects and we are converting them to proto files in the state view. Lets keep it consistent with that